docs/reference/cat/thread_pool.asciidoc

```diff
@@ -16,10 +16,11 @@ Which looks like:
 --------------------------------------------------
 node-0 analyze             0 0 0
 ...
```

Comment:
Hmm...I was thinking instead we should make the `...` translate to an optional pattern that would match the specified thread pool format (node-\d+ \S+ \d+ \d+ \d+). That way we do not have to require there be a thread pool between analyze and fetch_shard_started.